package.json

```diff
@@ -17,6 +17,7 @@
   },
   "devDependencies": {
     "@openmrs/esm-framework": "next",
+    "@openmrs/ngx-formentry": "next",
```

Comment:
This should not be a dev dependency of the workspace, just the module that needs it.